notebooks/examples/create_a_custom_suite.ipynb

```diff
@@ -0,0 +1,759 @@
+{
```

Comment:
<p>I don't think this is something we want to encourage to do... better to just directly refer to the API ref</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Yes this was written before we had it. and then I thought it's kind of convenient.</p><p>but will delete :)</p>